### DISCUSSION TITLE
求直播间快捷键

### DISCUSSION THREAD

**Discussion by Kal-tsit63** [the1812/Bilibili-Evolved#3325] [功能建议 / Ideas]
如标题，希望插件可以在直播间实现播放视频时的快捷键。如f，全屏等

**Comment by the1812** [comment]
看了下直播间的控制栏, b 站这个命名不知道该怎么定位全屏按钮, 只有个弹幕设置和清晰度是有名字的
![image](https://user-images.githubusercontent.com/26504152/210130715-d004c0f5-04c7-40e2-a4d9-f9ae5f7fbd51.png)
